README.md

```diff
@@ -47,6 +47,61 @@ Instructions
 9. Load the firmware onto the motor board:
 
 	$ stm32flash -w mcv4.bin -v /dev/ttyUSB<X>
+	
+## Controls
+
+The motor board is largely controlled over the serial interface. There is one physical push button on the board that is a firmware reset button.
+
+## USB interface
```

Comment:
I think this interface documentation would be better placed somewhere other than here. The readme should more describe what the repo is and how to use it, rather than the implementation specifics of how the project works. That's what docs are for, save this getting bloated.

Reply:
I spoke with @rgilton about this last night, and we agreed that the FW repos are the most appropriate place for these docs (and definitely better than j5).

Reply:
+1 to keeping the docs close to the code they document.

If there's a concern about bloating the README then we could perhaps have a DOCS.md or a docs/ directory or something, but I don't think the details matter a great deal beyond trying to be consistent between our firmware repos.